nipype/pipeline/engine/tests/test_engine.py

```diff
@@ -518,35 +509,26 @@ def func2(a):
     w1.config['execution'] = {'stop_on_first_crash': 'true',
                               'local_hash_check': 'false',
                               'crashdump_dir': wd}
-    error_raised = False
     # create dummy distributed plugin class
     from nipype.pipeline.plugins.base import DistributedPluginBase
 
     class RaiseError(DistributedPluginBase):
         def _submit_job(self, node, updatehash=False):
             raise Exception('Submit called')
-    try:
+
+    with pytest.raises(Exception) as excinfo:
```

Comment:
correct. most of nipype is ok - we have a couple of general exception catches, mostly because we don't know how python functions or external tools called via subprocess are going to crash